src/Security.tsx

```diff
@@ -33,7 +33,12 @@ const Security: React.FC<{
     return oktaAuth.authStateManager.getAuthState();
   });
   const [oktaAuthMajorVersion] = React.useState(() => {
-    const majorVersion = oktaAuth?.userAgent?.split('/')[1]?.split('.')[0];
+    if (!oktaAuth || !oktaAuth._oktaUserAgent) {
+      return null;
+    }
+
+    const oktaAuthVersion = oktaAuth._oktaUserAgent.getVersion();
+    const majorVersion = oktaAuthVersion?.split('/')[1]?.split('.')[0];
```

Comment:
getVersion returns version in `x.y.z` format. The parser can be adjusted to `oktaAuthVersion?.split('.')[0]`